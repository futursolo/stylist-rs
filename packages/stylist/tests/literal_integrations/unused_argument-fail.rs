fn main() {
    let _ = stylist::css! {r#"
        background: ${used};
    "#, unused = 1000, used = "black"
    };
}
