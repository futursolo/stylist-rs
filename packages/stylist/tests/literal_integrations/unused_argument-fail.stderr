error: argument unused is not used, arguments must be used
 --> $DIR/unused_argument-fail.rs:4:9
  |
4 |     "#, unused = 1000, used = "black"
  |         ^^^^^^
